documentation/src/main/asciidoc/user_guide/eviction.adoc

```diff
@@ -61,27 +55,43 @@ the configured `size`.  This type only works with primitive wrapper, String
 and byte[] types, thus if custom types are desired you must enable
 storeAsBinary.  Also MEMORY based eviction only works with LRU policy.
 
+==== Storage type
+
+Infinispan allows the user to configure in what form their data is stored.  Each
+form supports the same features of Infinispan, however evicttion can be limited
+for some forms.  There are currently three storage formats that Infinispan
+provides, they are:
+
+.`OBJECT`
+Stores the keys and values as is provided from the user.  Only `COUNT` eviction
```

Comment:
Stores the keys and values as objects. 

I would mention that this means on-heap